packages/mobile/src/features/videos/components/SingleScreen/DescriptionLeft.tsx

```diff
@@ -53,7 +57,7 @@ export class DescriptionLeft extends React.Component<any,DesLeft> {
               showsHorizontalScrollIndicator={false}
               horizontal={true}
               data={this.state.listVideos}
-              renderItem={({item}) => this.DesLeft(item)}
+              renderItem={({item}) => this.desLeft(item)}
```

Comment:
funkcja strzalkowa w props

Reply:
tak samo tutaj to nie mój kod, ale mogę przerobić tą funkcje